Update README.md (#8)
curveball/cors: README.md

app.use(cors());
```

When manually providing CORS options, this is how it should look:

> ```

Reviewer: Make sure this gets syntax highlighted as typescript.

app.use(cors({
    allowOrigin: '*',
    allowHeaders: ['Content-Type', 'Accept'],
    allowMethods: ['GET', 'POST'],
    exposeHeaders: ['Link', 'Date']
}));

```

> Providing CORS headers is optional, without providing any CORS options, it will auto-generate these values

Reviewer: ```suggestion
If no options are given, it will use these defaults:
```
